.github/integration/setup/common/22_create_and_encrypt_large_file.sh

```diff
@@ -17,3 +17,14 @@ touch empty.c4gh
 
 dd if=largefile.c4gh bs=1 count="$((15000+RANDOM))" of=truncated1.c4gh
 dd if=largefile.c4gh bs=1 count=10 of=truncated2.c4gh
+
+dd if=/dev/random of=wrongly_encrypted.raw count=1 bs=$(( 1024 * 1024 *  1 )) iflag=fullblock
+
+cat > wrong_key.pub <<EOF
+-----BEGIN CRYPT4GH PUBLIC KEY-----
+fGi6kvZlQN37PLmumygbNaSLf0NA+kj4KB1b70HFFCU=
+-----END CRYPT4GH PUBLIC KEY-----
+EOF
```

Comment:
Why not use crypth4gh to generate a new keypair and use that public key?

Reply:
I wanted to avoid the hassle of using passphrases when generating a new keypair.

Reply:
if you use the golang binary you can create the keypair like this:

```
crypt4gh generate -n c4gh -f wrong_key -p VoaagCP1YgAZeEyl2eJAkHv9lkcWXWFgm
```

Reply:
nice! I was not using the golang binary indeed.  But it [seems](https://github.com/neicnordic/sda-pipeline/blob/ad9fa3d0da3b6aa86979c94c5e6ccb36af910f66/.github/integration/setup/common/20_tools.sh#L3) that the tests don't either. So maybe keep the current solution atm?

Reply:
you can generate keys using that python `crypt4gh` as well https://github.com/neicnordic/sda-helm/blob/master/dev_tools/scripts/create-secrets.sh#L25-L35 - I am fine with both generating a new one or having it hardcoded.